core/src/main/scala/cats/data/Validated.scala

```diff
@@ -358,6 +358,12 @@ sealed abstract class Validated[+E, +A] extends Product with Serializable {
       case Invalid(e)   => Validated.invalidNel(e)
     }
 
+  def toValidatedA[F[_]: Applicative, EE >: E, AA >: A]: Validated[F[EE], AA] =
+    this match {
+      case v @ Valid(_) => v
+      case Invalid(e)   => Validated.invalidA(e)
+    }
```

Comment:
Similar to the previous comment, but this case looks even more controversial to me, since a user will have to specify both `A` and `B` types every time:
```
myvalidated.toValidatedA[SomeApplicative, LeftType, RightType]
```

Reply:
same issue here `E` is covariant while `F[_]` is not

Reply:
we can crop it to `def toValidatedA[F[_]: Applicative, EE >: E]: Validated[F[EE], A]` or use the partial application as well, what do you prefer?